docs/enterprise-guide/enterprise-on-premises-guide/support/installation-support/script/op-support-data.sh

```diff
@@ -5,12 +5,12 @@
 #VERBOSE LOGGING
 #set -x
 
-hash netcat 2>/dev/null
-if [ $? == 1 ]; then
-  echo
-  echo "Please install netcat (apt install netcat, yum install netcat) as this script depends on netcat. Thank you."
-  exit
-fi 
+#hash netcat 2>/dev/null
```

Comment:
If you don't need lines anymore, please just remove them rather than commenting them out. We can always still find them in the git history, if we should want them back.

Reply:
I removed all hashmarked lines,  correct we have an older version in github